packages/main/src/Link.js

```diff
@@ -25,7 +25,7 @@ const metadata = {
 		/**
 		 * Defines whether the component is disabled.
 		 * <br><br>
-		 * <b>Note:</b> When disabled, the component cannot be triggered by the user.
+		 * <b>Note:</b> When disabled, the click event cannot be triggered by the user.
```

Comment:
When disabled, the component doesn't react on a click event by the user.

Reply:
As discussed, I will not make change here.